content/Consumer Privacy Regulations CCPA and GDPR - Pablo Blanco.md

```diff
@@ -0,0 +1,240 @@
+Businesses have a track record of using personal information to benefit their
+agenda, CCPA will serve to protect California consumer rights and encourage
+stronger privacy, traceability, and greater transparency overall. CCPA allows
+consumers to have control, ownership, gain security over their personal
+information, and also the ability to request that any business disclose (and
+delete) the personal information collected.
+
+This data protection gives consumers the right to:
+
+- Know what personal information is being collected
+- Access the personal information that is collected, and request its deletion
+- Know if their personal information is being shared, and if so, with whom
+  opt-out of the sale of their personal information
+- Have equal service and price, whether or not they choose to exercise their
+  privacy rights (right to non-discrimination).
+
+It’s also prohibited selling the personal information of consumers ages 13–16
+(unless the consumer opts-in). For consumers under the age of 13, consent from a
+parent is a must.
+
+## Who does the CCPA apply to?
+
+CCPA defines a business as a for-profit entity that collects consumer personal
+data and applies to all companies that serve California residents, whether they
+are physically placed in California or not. If the company meets at least one of
+the following thresholds, may be subject to compliance:
+
+- Companies with at least \$25 million in annual revenue (applies to total
+  revenue, not just revenue in California).
+- Companies that annually buy, sell, receive, or share personal information of
+  at least 50,000 consumers, homes, or devices for commercial purposes.
+- Companies that derive 50% or more of their annual revenue from the sale of
+  personal information.
+
+## Some differences between CCPA and GDPR
+
+The European General Data Protection Regulation is an evolution of the EU’s
+existing data rules and replaced the Data Protection Directive (DPD). It
+addresses many of the lacks in the DPD, including adding requirements for
+documenting IT procedures, performing risk assessments under certain conditions,
+notifying the consumer and authorities when there is a breach, and strengthening
+rules for data minimization.
+
+The GDPR protects similar private data as the CCPA and recommends that companies
+provide a “reasonable” level of protection for personal information. It also
+mandates that the business explains to the customer how their information will
+be used and to explicitly ask for their permission to collect and process it.
+
+No matter how influenced the CCPA may have been by the GDPR, there are some
+clear differences we should notice in each legislation. Although the two laws
+are quite similar, the CCPA differs from the GDPR in a few important ways:
+
+- **Business impacted**: whereas businesses of any size must comply with the
+  GDPR, the CCPA only impacts businesses that reach a certain size that
+  processes a certain amount of data from consumers.
```

Comment:
```suggestion
  GDPR, the CCPA only impacts businesses that reach a certain size and/or that
  process a certain amount of data from consumers.
```
Seems to be something missing from that sentence, could an `and` or `or` make it right?